lib/core/src/Cardano/Wallet/Primitive/Types/UTxOIndex/Internal.hs

```diff
@@ -416,6 +420,9 @@ data InvariantStatus
     | InvariantAssetsInconsistent
       -- ^ Indicates that the 'index' and the cached 'balance' value disagree
       --   about which assets are included.
+    | InvariantIndexedCoinsAreNotCoins
+      -- ^ Indicates that all inputs indexed as 'coins' are indeed ada-only
+      -- outputs.
```

Comment:
Hi @KtorZ

This comment is still wrong, I think, because it indicates the opposite of what it says.

Suggested wording (though feel free to pick a different wording):

```suggestion
    | InvariantIndexedCoinsAreNotCoins
      -- ^ Indicates that at least one member of 'coins' is not ada-only.
```

Reply:
Fixed :+1: